include/dynamic_graph_bridge/fwd.hh

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright CNRS 2021
+ *
+ * Author: Florent Lamiraux
+ *
+ * This file is part of sot-core.
+ */
+
+#ifndef DYNAMIC_GRAPH_BRIDGE_FWD_HH
+#define DYNAMIC_GRAPH_BRIDGE_FWD_HH
+
+#include <dynamic-graph/python/fwd.hh>
+
+namespace dynamicgraph {
+  // classes defined in sot-core
+  class Interpreter;
+  typedef shared_ptr<Interpreter> InterpreterPtr_t;
```

Comment:
Just a nickpick, but as we are here why not modifying dynamic-python to add a type 
`dynamicgraph::Interpreter::SharedPtr`?
This allows to narrow done the scope of the type, and avoid an explosion of header files.